model/Core/Classes/SoftwareAgent.md

```diff
@@ -0,0 +1,20 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# SoftwareAgent
+
+## Summary
+
+A software agent.
+
+## Description
+
+A SoftwareAgent is a software representation of a user that could be assigned authorization to perform tasks on behalf of a person or organization.
```

Comment:
Suggest changing to: "A SoftwareAgent is a software program with the potential to act on behalf of a user (person or organization) when it possesses appropriate authority."

The word representation has potential confusion of being interpreted as a facsimile. Really the software agent is acting "on behalf of" the user.

Reply:
From discussion in meeting: 

A SoftwareAgent is a software program that is given the authority (similar to a user's authority) to perform an operation.